client/verta/verta/endpoint/build/_build.py

```diff
@@ -110,6 +111,19 @@ def get_scan(self) -> _build_scan.BuildScan:
         """
         return _build_scan.BuildScan._get(self._conn, self.id)
 
+    def get_hardware_compatibility(self):
+        """Get this build's hardware compatibility. If no hardware compatibility was specified,
+        returns None.
+
+        .. versionadded:: 0.25.0
+
+        Returns
+        -------
+        :class:`~verta.endpoint.build.BuildHardwareCompatibility` or None
+
+        """
+        return BuildHardwareCompatibility._get(self._conn, self.id)
```

Comment:
I think we should implement this as

```python
@property
def hardware_compatibility(self):
```

and construct `BuildHardwareCompatibility` using just `self._json`. A build's hardware compatibility won't change after it's created, right? If so we can just base it on the data we already fetched